packages/tooltip/components/TooltipContent.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from "react";
+import { cx } from "emotion";
+import { tooltip, getTooltipArrow } from "../style";
+import { darkMode, padding } from "../../shared/styles/styleUtils";
+import { BaseTooltipProps } from "./Tooltip";
+import { Direction } from "../../dropdownable/components/Dropdownable";
+
+export interface TooltipContentProps extends BaseTooltipProps {
+  direction?: Direction;
```

Comment:
This is not actually optional, but it's passed via `React.cloneElement` from Dropdownable.

Anybody have any ideas how have proper type safety in this situation? I haven't been able to find a good solution